Source/Fluxor.Blazor.Web.ReduxDevTools/Fluxor.Blazor.Web.ReduxDevTools.csproj

```diff
@@ -24,7 +24,6 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.Components" Version="3.1.2" />
     <PackageReference Include="Microsoft.AspNetCore.Components.Web" Version="3.1.2" />
-    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
```

Comment:
Newtonsoft is needed. System.Text.Json won't handle many cases (such as dictionaries).

When submitting PRs it's best to only do one thing in the PR. For example, if this PR had only updated dependency versions then I had approved it, but I can't accept it as-is because it has some changes I don't agree with.